Readme.md

```diff
@@ -99,13 +99,15 @@ Default is `%`.
 #### Extend
 
 Extends the width of the string by the specified percentage. Useful if you will be localizing into languages such as German which can be 30% longer than English.
+We extend it by a set of predefined Asian characters.
```

Comment:
They're not all Asian characters. How about:
`We extend the string by randomly selecting characters from a predefined array of non-ASCII characters`